docker/dev.Dockerfile

```diff
@@ -1,3 +1,6 @@
+# Importing Typst official Image
+FROM ghcr.io/typst/typst:v0.11.0 as typstOfficialImage
```

Comment:
Can we install this within the old image?  If its not doable now, lets create a backlog ticket for this.